client/src/component/filmStats.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+// TODO host tomatos on s3
+let num = 3.5;
```

Comment:
What's this for?

Reply:
oops that's leftover from some early tests